src/llama_recipes/utils/train_utils.py

```diff
@@ -98,10 +104,13 @@ def train(model, train_dataloader,eval_dataloader, tokenizer, optimizer, lr_sche
                         optimizer.step()
                         optimizer.zero_grad()
                         pbar.update(1)
+                if tracker is not None and track_metrics:
+                    tracker.track(loss, name='loss', stage='train')
```

Comment:
should we be gathering loss from all ranks in the fsdp case?

Reply:
also, can we track throughput (tokens per second and/or tokens per step) during training?

Reply:
The original code tracked only on rank 0 in fsdp. 
If we need to track on all ranks in Aim, we can do that.